### PR TITLE
Web API: Retrieve evaluated article from EuropePMC

### PR DESCRIPTION
**Pull request by de-code**
part of https://github.com/elifesciences/data-hub-issues/issues/771

requires https://github.com/elifesciences/data-hub-core-airflow-dags/pull/1281

This should retrieve the all of the evaluated articles from EuropePMC.
We only need the DOI really (but the WebAPI doesn't seem to support a field selector yet).

### CODE REVIEW

File: kustomizations/apps/data-hub-configs/web-api-data-pipeline.config.yaml [hunk 1]
```diff
@@ -370,3 +370,19 @@ webApi:
       recordTimestamp:
         itemTimestampKeyFromItemRoot:
           - date
+  # EuropePMC: Articles that are evaluated by Sciety (via LabsLink)
+  - dataset: '{ENV}'
+    table: 'europepmc_sciety_evaluated'
```
Comment (de-code) [inline kustomizations/apps/data-hub-configs/web-api-data-pipeline.config.yaml:375]: Or maybe `europepmc_sciety_evaluated_via_labslink` to make it more explict?

Comment (HazalCiplak) [reply kustomizations/apps/data-hub-configs/web-api-data-pipeline.config.yaml:375]: It would be good, if there is another way to evaluate.

**Review by HazalCiplak — APPROVED** [review]

File: kustomizations/apps/data-hub-configs/web-api-data-pipeline.config.yaml (hunk 1 above)
Comment (de-code) [reply kustomizations/apps/data-hub-configs/web-api-data-pipeline.config.yaml:375]: LabsLink is how we get the evaluations into EuropePMC. But they also started integrating DocMaps.

**Comment by de-code** [comment]
https://github.com/elifesciences/data-hub-core-airflow-dags/pull/1281 has now been deployed (via `v0.0.139`)

**Comment by de-code** [comment]
Config changes deployed.

```bash
kubectl --namespace=data-hub exec data-hub--prod-worker-0 -- tail -20 /dag_config_files/web-api-data-pipeline.config.yaml
```

**Pull request merged**